src/index.html

```diff
@@ -7,6 +7,462 @@
+            <a class="nav__link is-active" href="#">
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              СМАРТФОНЫ И ТЕЛЕФОНЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ГАДЖЕТЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ПЛАНШЕТЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ФОТО
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+
+    <main class="main">
+
+      <div class="card" data-qa="card">
+
+        <div class="card__img">
+          <img
+            src="./images/imac.jpeg"
+            alt="prodact_imac"
+            class="card__img-product">
+        </div>
+
+        <div class="card-info-title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+
+        <div class="card-info-number">
+          Код товара: 195434
+        </div>
+
+        <rating class="card__rating">
+          <div class="card__rating-stars rating-stars rating-stars--4">
+            <div class="rating-stars__star"></div>
+            <div class="rating-stars__star"></div>
+            <div class="rating-stars__star"></div>
+            <div class="rating-stars__star"></div>
+            <div class="rating-stars__star"></div>
+          </div>
+
+          <div class="card__rating-comments">
+            Отзывов: 5
+          </div>
+        </rating>
+
+        <div class="card__price">
+          <div class="card__price-text">
+            Цена:
+          </div>
+          <div class="card__price-sum">
+            69 999 грн
+          </div>
+        </div>
+        <div class="card__buy">
+          <a
+            class="card__buy-link"
+            href="#"
+            data-qa="card-hover"
+          >
+            КУПИТЬ
+          </a>
+        </div>
+      </div>
+
+
+      <div class="card">
+        <div class="card__img">
+          <img
+            src="./images/imac.jpeg"
+            alt="prodact_imac"
+            class="card__img-product">
+        </div>
+
+        <div class="card-info-title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+
+        <div class="card-info-number">
```

Comment:
same here. Make BEM-block only if it's an independent block that can be reused in other places